src/main/java/org/jenkins/tools/test/model/MavenBom.java

```diff
@@ -47,10 +50,8 @@ public File writeFullDepPom(File workDir) throws IOException {
         try (FileOutputStream out = new FileOutputStream(fullDepPom)) {
             writer.write(out, modified);
             return fullDepPom;
-        } catch (FileNotFoundException e) {
-            e.printStackTrace();
         } catch (IOException e) {
-            e.printStackTrace();
+            LOGGER.log(Level.WARNING, "Failed to write full dependency POM; continuing", e);
```

Comment:
`FileNotFoundException` is an `IOException` so there was no need for the preceding `catch` clause.